lib/wrapper.pm

```diff
@@ -0,0 +1,57 @@
+# SUSE's openQA tests
+#
+# Copyright 2023 SUSE LLC
+# SPDX-License-Identifier: FSFAP
+#
+# Summary: Wrapper for package installation, zypper or transactional update
+# Maintainer: QE Core <qe-core@suse.de>
+
+package wrapper;
+use Mojo::Base qw(Exporter);
+use testapi;
+use version_utils qw(is_transactional);
+use transactional qw(trup_call check_reboot_changes);
+use utils qw(zypper_call);
+
+our @EXPORT = "install_package";
+
+=head1 DESCRIPTION
+
+    Wrapper for zypper_call and trup_call
+
+=head2 install_package
+
+    install_package(packages => 'kernel-devel' [, trup_packages => 'kernel-default-base] [, zypper_packages => 'kernel-devel']);
+
+C<packages> defines packages to install for both zypper and trup call,
+different packages can be installaed on transactional system via C<trup_packages>
+or on standard system via C<zypper_packages>
+After trup_call check_reboot_changes can be executed via C<trup_check_reboot> parameter
+
+=cut
+
+sub install_package {
+    my (%args) = @_;
+    my $trup_packages = $args{trup_packages} // $args{packages};
+    my $zypper_packages = $args{zypper_packages} // $args{packages};
+    die 'At least one of packages, trup_packages, zypper_packages arguments is required' unless defined($args{packages} || $args{trup_packages} || $args{zypper_packages});
+    my $timeout = $args{timeout} // 500;
```

Comment:
First, why add a new file? This should go to `utils.pm` or at least `qam.pm` because it'll be used a lot.

Second, I really don't like that all packages have to be passed in a keyword argument. I'd prefer the package list handling like this:

    my ($packages, %args) = @_;

    if (is_transactional) {
        $packages .= ' ' . $args{trup_extra} // '';
    }
    else {
        $packages .= ' ' . $args{zypper_extra} // '';
    }

So that I can call just `install_package('foo');`.

Reply:
I think I read somewhere it should be in separated package, but I don't mind to have it in likely `utils.pm`.
Sure,  I will do the packages default parameter without keyword argument.

Reply:
I changed the code, if nobody disagrees @czerw @foursixnine ? I would move the install_package into utils.

Reply:
> This should go to utils.pm or at least qam.pm because it'll be used a lot.

That it will be used a lot isn't a good reason to keep making lib/utils grow, so @dzedro lets keep it separate.

In the end, Jozef's work would end up as a stepping stone to move towards something that abstracts most things from the caller

Reply:
OK, can we at least rename the file so that it sounds like something related to package installation? `rpm.pm`, `package_utils.pm`, or something like that...